services/web/server/src/simcore_service_webserver/socketio/messages.py

```diff
@@ -51,6 +51,7 @@ async def _safe_emit(
             room=room,
             ignore_queue=ignore_queue,
         )
+        _logger.info("emitted socketio event '%s' to room '%s'", event, room)
```

Comment:
you really want this a info!? I think this is going to be very loud. i would recommend DEBUG

Reply:
I've inherited the log leve from what was previously there. I think it's like this